vehicle_status_generator/package.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0"?>
+
+<!--  
+ Copyright (C) 2018-2021 LEIDOS.
+
+ Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ use this file except in compliance with the License. You may obtain a copy of
+ the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ License for the specific language governing permissions and limitations under
+ the License.
+-->
+
+<package format="3">
+  <name>vehicle_status_generator</name>
+  <version>3.3.0</version>
+  <description>This generates BSM according to J2735 standard.</description>
```

Comment:
Same with this description, this should only be generating the mobility operations message needed for the all-way-stop use case right?